codemon/codemon.py

```diff
@@ -41,3 +41,6 @@ def main():
         contestName = sys.argv[countArg]
         practiceFiles = get_practice_files()
         init(contestName, practiceFiles)
+
+      elif arg == "help":
+        showHelpDetailed()
```

Comment:
You misunderstood the issue, we had to introduce a flag `--help` which would display that simple `showHelp()`.